.github/workflows/on-cron.yml

```
@@ -0,0 +1,18 @@
+name: "Close / Open milestones automatically"
+on:
+  schedule:
+    - cron: "0 0 * * *"
```

Once a day might be too much of a delay

```suggestion
    - cron: "0 * * * *"
```

---

Yea